### ISSUE TITLE
Accuracy in validation set does not improve

### ISSUE THREAD

**Issue by BirkhoffG** [BirkhoffG/pnet_pytorch#3]
The validation accuracy and loss do not change during training. It seems like the network does not train after the first epoch. But I try different batch size and learning rate, and I always stuck at acc=67.48466257668711%. 

```
❯ python -m src.main --batch-size 128 -lr 0.01
loading data...
building vocabulary...
[epoch=0] loss: -0.0015659998322034698, acc: 0.0%
100%|███████████████████████████████████████████████████████████████████████████████████| 72/72 [00:29<00:00,  2.42it/s]
[epoch=1] loss: -0.005321807343117716, acc: 67.48466257668711%
100%|███████████████████████████████████████████████████████████████████████████████████| 72/72 [00:27<00:00,  2.61it/s]
[epoch=2] loss: -0.005321807343117716, acc: 67.48466257668711%
```
